avr-hal-generic/src/i2c.rs

```diff
@@ -201,7 +206,8 @@ macro_rules! impl_twi_i2c {
                 data: $twdr:ident,
             },
         }
-    ) => {
+    ) => {$crate::paste::paste! {
+        /// I2C Master
         $(#[$i2c_attr])*
         pub struct $I2c<CLOCK: $crate::clock::Clock, M> {
```

Comment:
While we're at it, should we rename this one to `[< $I2c Master>]`?  Maybe we should check what other HALs that support both modes, are naming them ...

Reply:
Cool, I was thinking the same thing. I'll look around and see what's common.